unstructured/metrics/evaluate.py

```diff
@@ -208,18 +214,30 @@ def group_text_extraction_accuracy(
             f"Data cannot be aggregated by `{grouping}`."
             f" Check if it's empty or the column is missing/empty."
         )
-    grouped_acc = (
-        df.groupby(grouping)
-        .agg({"cct-accuracy": [_mean, _stdev, "count"]})
-        .rename(columns={"_mean": "mean", "_stdev": "stdev"})
-    )
-    grouped_miss = (
-        df.groupby(grouping)
-        .agg({"cct-%missing": [_mean, _stdev, "count"]})
-        .rename(columns={"_mean": "mean", "_stdev": "stdev"})
-    )
-    grouped_df = _format_grouping_output(grouped_acc, grouped_miss)
-    _write_to_file(export_dir, f"all-{grouping}-agg-cct.tsv", grouped_df)
+    if metric_strategy == "text_extraction":
+        grouped_acc = (
+            df.groupby(grouping)
+            .agg({"cct-accuracy": [_mean, _stdev, _count]})
+            .rename(columns={"_mean": "mean", "_stdev": "stdev", "_count": "count"})
```

Comment:
renaming is repeated 3 times, i think we can do it just before writing the df to file.